opencti-platform/opencti-graphql/src/connector/importCsv/importCsv-connector.ts

```diff
@@ -112,14 +109,9 @@ const initImportCsvConnector = () => {
 
   return {
     start: async () => {
-      logApp.info(`[OPENCTI-MODULE] Starting ${connectorConfig.name} manager`);
-
       const context = executionContext(connectorConfig.id.toLowerCase());
-      // Register connector queues if not exists
-      const cbError = async (error: any) => {
-        if (error) await registerConnectorQueues(connector.id, connector.name, connector.connector_type, connector.connector_scope);
-      };
-      await existsConnectorQueues(connector.id, cbError);
+      logApp.info(`[OPENCTI-MODULE] Starting ${connectorConfig.name} manager`);
+      await registerConnectorQueues(connector.id, connector.name, connector.connector_type, connector.connector_scope);
```

Comment:
I didn't understand that you could call this method even if the queue was already present